CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ set(CMAKE_SKIP_INSTALL_RPATH OFF)
 set(CMAKE_SKIP_RPATH OFF)
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
+set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
Now I see a problem with this line :) 

It should be placed after this block: https://github.com/eclipse/xacc/blob/master/CMakeLists.txt#L102-L106
(`CMAKE_INSTALL_PREFIX` is not set to a proper default value yet at this point. Our CI scripts all use this default hence didn't set it explicitly)

Reply:
I see. I've just moved the line. Thanks!